packages/calcite-components/src/components/autocomplete-item/autocomplete-item.scss

```diff
@@ -0,0 +1,90 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-autocomplete-background-color: Specifies the background color of the component.
+ * @prop --calcite-autocomplete-background-color-hover: Specifies the background color of the component when hovered.
```

Comment:
@alisonailea do we really need this var? I thought if a user could use the host to set the hover state bg color a var wasn't necessary?

Reply:
Like:

```
calcite-autocomplete-item:hover{
--calcite-autocomplete-background-color: red;
}
```

Reply:
@macandcheese do you know? I thought we only needed bg state specific vars when using the host selector wouldn't work?

Reply:
I believe what you have above (`calcite-autocomplete-item:hover {}`) is the recommendation, based on this at least: https://github.com/Esri/calcite-design-system/wiki/tokens-component#stateful-naming-rules

If we added separately focusable / clickable areas to an `autocomplete-item`, then we'd need the explicitly defined stateful tokens.

Reply:
@alisonailea can you confirm?